ui/src/components/credentials/CredentialDetails.tsx

```diff
@@ -1,4 +1,4 @@
-import { Button, Card, Space, Typography } from "antd";
+import { Button, Card, Col, Grid, Row, Space, Typography } from "antd";
```

Comment:
The page credential details has some room for improvement

<img width="486" alt="image" src="https://github.com/0xPolygonID/issuer-node/assets/15896177/f142fc32-61d1-472c-8281-62211c3d9977">

  * The buttons Revoke and Delete could probably be inline on this screen, but if the stack, I'm missing some margin.
  * The Credential Details box could look better. We have a similar issue with this box in Schema Details. See below:

<img width="485" alt="image" src="https://github.com/0xPolygonID/issuer-node/assets/15896177/e55fe707-9132-4c44-965e-c7e3597a6d3e">

Reply:
This is now working better but I see that something breaks on resize:

https://github.com/0xPolygonID/issuer-node/assets/15896177/0a43793c-dbf4-45c6-a8c8-20b698b6e493

Reply:
Per my tests, commenting [this line](https://github.com/0xPolygonID/issuer-node/blob/PID-14-responsive-ui/ui/src/components/shared/Detail.tsx#L30) fixes the issue.

Reply:
Regarding the text not recovering its original size, [there's a bug opened in antd](https://github.com/ant-design/ant-design/issues/32899).

Reply:
Yep, it looks definitely related. If I remove the copyable prop from the `Typography.Text`, I get no exceptions and no resize problems. We should consider implementing our own copy button. What a crappy lib.

Reply:
Uff since v4, so what is your idea? Follow in a new issue?

Reply:
I'd stop using the copy feature of antd and implement our own.

Reply:
You're on holiday man... don't you prefer to hand off this PR? I can take it from here if you agree.

Maybe we can also draft it until you're back. I don't think this is urgent. What do you think @cneuro?

Reply:
It's not a new bug, we can fix it later properly. issue: https://polygonid.atlassian.net/browse/PID-735?atlOrigin=eyJpIjoiNGRiOTYzNjkzYmJjNGIzNDgwNGU4NWJlNWNmYjY0ZDIiLCJwIjoiaiJ9

Reply:
@cneuro [has decided](https://0xpolygon.slack.com/archives/C044J4HN360/p1684253238191329?thread_ts=1684251872.147749&cid=C044J4HN360) to create [a new issue](https://polygonid.atlassian.net/browse/PID-735) to fix this error.